logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnConfigRequest.kt

```diff
@@ -0,0 +1,43 @@
+package com.wire.kalium.logic.feature.call.scenario
+
+import com.sun.jna.Pointer
+import com.wire.kalium.calling.Calling
+import com.wire.kalium.calling.callbacks.CallConfigRequestHandler
+import com.wire.kalium.calling.types.Handle
+import com.wire.kalium.logic.callingLogger
+import com.wire.kalium.logic.data.call.CallRepository
+import com.wire.kalium.logic.feature.call.AvsCallBackError
+import com.wire.kalium.logic.feature.call.CallManagerImpl
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+//TODO create unit test
+class OnConfigRequest(
+    private val calling: Calling,
+    private val callRepository: CallRepository
```

Comment:
```suggestion
    private val callRepository: CallRepository,
    private val callingScope: CoroutineScope
```

Maybe we can tackle this now before moving forward?

For testing, we can pass `TestScope`.